pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "gnmi-proto"
-version = "0.1.0a5"
```

Comment:
a5 is not released yet, so this bump is not required

Reply:
Right! I dropped the commit. Wasn't sure about the versioning yet, but just saw that it is handled via tags.